docs/tips.md

```diff
@@ -184,11 +184,12 @@ run  = 'hyprctl hyprpaper reload ,"$0"'
 for  = "linux"
 desc = "Set as wallpaper"
 
-# Linux: Sway + Swaybg
+# Linux: Swaybg
 [[opener.set-wallpaper]]
-run  = 'cp "$0" "$HOME/.config/sway/wallpaper" && killall swaybg && swaymsg exec "swaybg -i $HOME/.config/sway/wallpaper"'
+run  = 'killall swaybg; swaybg -m fill -i "$0"'
 for  = "linux"
 desc = "Set as wallpaper"
+orphan = true
```

Comment:
Thank you for the patch!

This seems unnecessary, as after setting `orphan = true` the user wouldn't be able to see the shell command running in the task manager even when the command fails